### PR TITLE
DOCSP-31860-binary-constructors-shell

### PR DESCRIPTION
**Pull request by jason-price-mongodb**
## DESCRIPTION

Shell updates for new constructors.

## STAGING

https://docs-mongodbcom-staging.corp.mongodb.com/mongodb-shell/docsworker-xlarge/DOCSP-31860-binary-constructors-shell/changelog/#v2.0.0

## JIRA

https://jira.mongodb.org/browse/DOCSP-31860

## BUILD LOG

https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=64e4e67c57e37806c5446d69

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [ ] Is this free of spelling errors?
- [ ] Is this free of grammatical errors?
- [ ] Is this free of staging / rendering issues?
- [ ] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

File: source/changelog.txt [hunk 1]
```diff
@@ -17,6 +17,20 @@ v2.0.0
...
+
+- ``mongosh`` returns binary values as ``Binary.createFromBase64(
+  <base64String> )`` values instead of ``Binary( Buffer.from(
+  <base64String> ) )`` values. For example:
+
+  - Starting in 2.0: ``binaryValue:
+    Binary.createFromBase64("SGVsbG8gV29ybGQhCg==")``
+  - Earlier versions: ``binaryValue:
+    Binary(Buffer.from("SGVsbG8gV29ybGQhCg==", "hex"), 0)``
+
```
Comment (sarah-olson-mongodb) [inline source/changelog.txt:33]: (linking)
Consider adding a link to where folks can get the additional information. I know this probably requires a follow on update, once things are linkable across repos but in this case I think it's worth the effort.

**Review by sarah-olson-mongodb — APPROVED** [review]
Thanks @jason-price-mongodb! This looks great. LGTM with one suggestion to follow up on after the merge.

**Comment by jason-price-mongodb** [comment]
@addaleax please review. Thanks.

File: source/changelog.txt [hunk 2]
```diff
@@ -17,6 +17,22 @@ v2.0.0
 
 *Upcoming*
 
+- Added support for these constructors:
+
+  - :method:`Binary.createFromBase64()`
```
Comment (addaleax) [inline source/changelog.txt:22]:
```suggestion
  - :method:`Binary.createFromBase64()`
  - :method:`Binary.createFromHexString()`
  - :method:`ObjectId.createFromBase64()`
```

**Comment by jason-price-mongodb** [comment]
@addaleax please re-review. Thanks.

File: source/changelog.txt [hunk 3]
```diff
@@ -17,6 +17,24 @@ v2.0.0
...
+  - ``ObjectId.createFromBase64()``
+
+- ``mongosh`` returns binary values as ``Binary.createFromBase64(
+  <base64String> )`` values instead of ``Binary( Buffer.from(
+  <base64String> ) )`` values. For example:
+
+  - Starting in 2.0.0: ``binaryValue:
+    Binary.createFromBase64("SGVsbG8gV29ybGQhCg==")``
+  - Earlier versions: ``binaryValue:
+    Binary(Buffer.from("SGVsbG8gV29ybGQhCg==", "hex"), 0)``
```
Comment (addaleax) [inline source/changelog.txt:34]: Same comment as https://github.com/10gen/docs-mongodb-internal/pull/4411#discussion_r1304236034  applies here as well – this doesn’t look like current/1.x mongosh output.

**Review by addaleax — APPROVED** [review]

**Comment by GaurabAryal** [comment]
LGTM!

**Review by GaurabAryal — APPROVED** [review]

**Pull request merged**